apps/institution-service/cypress.config.ts

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from "cypress";
+import "dotenv/config";
+
+export default defineConfig({
+  env: {
+    PORT: process.env.PORT || 8088,
```

Comment:
Why don't we just hardcode a port?

Reply:
Because I took this from the auth service, and that's how it was done there.

Reply:
Oh, actually, that's not true. I'll hard-code it.